Source/Core/DolphinQt2/Config/Graphics/GraphicsWindow.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Config/Graphics/GraphicsWindow.h"
+#include "DolphinQt2/Config/Graphics/AdvancedWidget.h"
+#include "DolphinQt2/Config/Graphics/EnhancementsWidget.h"
+#include "DolphinQt2/Config/Graphics/GeneralWidget.h"
+#include "DolphinQt2/Config/Graphics/HacksWidget.h"
+
+#include <QDialogButtonBox>
+#include <QTabWidget>
+#include <QVBoxLayout>
+
+GraphicsWindow::GraphicsWindow(QWidget* parent, X11Utils::XRRConfiguration* xrr_config)
+    : QDialog(parent), m_xrr_config(xrr_config)
+{
+  CreateMainLayout();
+  ConnectWidgets();
+
+  setWindowTitle(tr("Graphics"));
+  setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+}
+
+void GraphicsWindow::CreateMainLayout()
+{
+  auto* main_layout = new QVBoxLayout();
+  m_tab_widget = new QTabWidget();
+  m_button_box = new QDialogButtonBox(QDialogButtonBox::Ok);
+
+  main_layout->addWidget(m_tab_widget);
+  main_layout->addWidget(m_button_box);
+
+  auto* general_widget = new GeneralWidget(this, m_xrr_config);
```

Comment:
It's not about parenting (for most of the widgets atleast, for some the criticism is okay and can be changed), it's about being able to connect to the BackendChange signal of the Window